src/styles/settings/_colors.scss

```diff
@@ -238,6 +238,9 @@ $WEB_EXTENDED_SECONDARY_COLOR: #e51661;
 $WEB_EXTENDED_TERTIARY_COLOR: #6001ff;
 $WEB_VITALS_COLOR: #2962ff;
 
+// ASIDE
+$UPDATE_ASIDE_COLOR: $WEB_PRIMARY_COLOR_600;
```

Comment:
If you tie this variable to $WEB_PRIMARY_COLOR_600, then there's no point in adding additional variable anymore. In your solution, if WEB_PRIMARY_COLOR changes, so does the Aside color (and we want to avoid this). Please copy the # color code instead.